content/tutorials/publish-subscribe.textile

```diff
@@ -16,6 +16,56 @@ Messages published can contain string, JSON object, JSON array or binary data pa
 
 h2. Step 2 - Install Ably
 
+blang[java].
+  To start using Ably in your Android app, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your module-level @gradle.build@ file.
```

Comment:
I suspect we should perhaps split up Step 2 - install Ably into two steps:

Step 2a - Set up Android environment and project
Step 2b - Install Ably

If we do that then we can keep the same shared headings for all tutorials as having a different number of steps may be tricky if we aim to share the code for this tutorial across languages that don't need this set up stage.

BTW. You can have headings that change based on language, you can use a format like:

```
h2
  default: Step 2 - Install Ably
  java: Step 2 - Set up environment and install Ably
```